pkg/fleet-manager/plugin/plugin.go

```diff
@@ -32,25 +32,29 @@ import (
 )
 
 const (
-	MetricPluginName          = "metric"
-	GrafanaPluginName         = "grafana"
-	KyvernoPluginName         = "kyverno"
-	BackupPluginName          = "backup"
-	StorageOperatorPluginName = "storage-operator"
-	ClusterStoragePluginName  = "cluster-storage"
-	FlaggerPluginName         = "flagger"
-	PublicTestloaderName      = "testloader"
-
-	ThanosComponentName        = "thanos"
-	PrometheusComponentName    = "prometheus"
-	GrafanaComponentName       = "grafana"
-	KyvernoComponentName       = "kyverno"
-	KyvernoPolicyComponentName = "kyverno-policies"
-	VeleroComponentName        = "velero"
-	RookOperatorComponentName  = "rook"
-	RookClusterComponentName   = "rook-ceph"
-	FlaggerComponentName       = "flagger"
-	TestloaderComponentName    = "testloader"
+	MetricPluginName             = "metric"
+	GrafanaPluginName            = "grafana"
+	KyvernoPluginName            = "kyverno"
+	BackupPluginName             = "backup"
+	StorageOperatorPluginName    = "storage-operator"
+	ClusterStoragePluginName     = "cluster-storage"
+	FlaggerPluginName            = "flagger"
+	PublicTestloaderName         = "testloader"
+	SubMarinerBrokerPluginName   = "submariner-broker"
+	SubMarinerOperatorPluginName = "submariner-operator"
+
+	ThanosComponentName             = "thanos"
+	PrometheusComponentName         = "prometheus"
+	GrafanaComponentName            = "grafana"
+	KyvernoComponentName            = "kyverno"
+	KyvernoPolicyComponentName      = "kyverno-policies"
+	VeleroComponentName             = "velero"
+	RookOperatorComponentName       = "rook"
+	RookClusterComponentName        = "rook-ceph"
+	FlaggerComponentName            = "flagger"
+	TestloaderComponentName         = "testloader"
+	SubMarinerBrokerComponentName   = "sm-broker"
+	SubMarinerOperatorComponentName = "sm-operator"
```

Comment:
ditto

Reply:
Although I can change the targetNamespace to be shorter (i.e. , `submariner), the namespace in operator fixed at [`submariner-operator`](https://github.com/submariner-io/submariner-charts/blob/cdd86cf19f1f6fc1c6791c51d1e2bf09c479ee36/submariner-operator/templates/submariner.yaml#L5)`, and submariner-operator-submariner-operator-kurator-member1` is also too long.